gradle/libs.versions.toml

```diff
@@ -8,13 +8,17 @@ androidx-appcompat = "1.6.1"
 androidx-constraintlayout = "2.1.4"
 androidx-core-ktx = "1.12.0"
 androidx-espresso-core = "3.5.1"
+androidx-lifecycle-lifecycleViewmodelKtx = "2.3.0"
+androidx-lifecycleViewmodelKtx = "2.4.0"
```

Comment:
we usually keep one variable per lib. I don't see a scenario where we would want to keep 2 different versions of the same lib in the same project. this can cause bugs in runtime.

Reply:
i removed them from Gradle, but not sure why they were all kept here.

Reply:
Done